starknet_py/net/schemas/rpc.py

```diff
@@ -964,6 +964,11 @@ def make_dataclass(self, data, **kwargs) -> DeployAccountTransactionTrace:
 
 
 class L1HandlerTransactionTraceSchema(Schema):
+    # TODO (#1323): Explain with starknet, because spec doesn't contain execution_resources
```

Comment:
we can probably remove this comment/issue, as Ariel already admitted this to be a bug in spec that will be fixed soon

Reply:
I am not sure how they will change L1HandlerTransactionTraceSchema. execution_resources can be required or optional. IMO is better to wait for space change